app/viewModels/DepartureStatusViewModel.scala~HEAD

```diff
@@ -64,57 +63,62 @@ object DepartureStatus {
+      DepartureStatusViewModel("departure.status.releasedForTransit", actions = Seq(downloadTADAction(departure)))
   }
 
-  private def transitDeclarationRejected(config: FrontendAppConfig): DepartureStatusViewModel = {
-    case departure if departure.status == "TransitDeclarationRejected" =>
-      DepartureStatus(
+  private def transitDeclarationRejected(config: FrontendAppConfig): PartialFunction[Departure, DepartureStatusViewModel] = {
+    case departure if departure.status == TransitDeclarationRejected =>
+      DepartureStatusViewModel(
         "departure.status.transitDeclarationRejected",
         actions = Seq(ViewMovementAction(config.departureFrontendDeclarationFailUrl(departure.departureId), "viewDepartures.table.action.viewErrors"))
       )
   }
 
-  private def departureDeclarationReceived: DepartureStatusViewModel = {
-    case departure if departure.status == "DepartureDeclarationReceived" =>
-      DepartureStatus("departure.status.departureDeclarationReceived", actions = Nil)
+  private def departureDeclarationReceived: PartialFunction[Departure, DepartureStatusViewModel] = {
+    case departure if departure.status == DepartureDeclarationReceived =>
+      DepartureStatusViewModel("departure.status.departureDeclarationReceived", actions = Nil)
   }
 
-  private def guaranteeValidationFail(config: FrontendAppConfig): DepartureStatusViewModel = {
-    case departure if departure.status == "GuaranteeNotValid" =>
-      DepartureStatus("departure.status.guaranteeValidationFail", actions = Seq(viewGuaranteeValidationFailAction(departure.departureId, config)))
+  private def guaranteeValidationFail(config: FrontendAppConfig): PartialFunction[Departure, DepartureStatusViewModel] = {
+    case departure if departure.status == GuaranteeNotValid =>
+      DepartureStatusViewModel("departure.status.guaranteeValidationFail", actions = Seq(viewGuaranteeValidationFailAction(departure.departureId, config)))
   }
 
-  private def transitDeclarationSent: DepartureStatusViewModel = {
-    case departure if departure.status == "TransitDeclarationSent" =>
-      DepartureStatus("departure.status.transitDeclarationSent", actions = Nil)
+  private def transitDeclarationSent: PartialFunction[Departure, DepartureStatusViewModel] = {
+    case departure if departure.status == TransitDeclarationSent =>
+      DepartureStatusViewModel("departure.status.transitDeclarationSent", actions = Nil)
   }
 
-  private def writeOffNotification: DepartureStatusViewModel = {
-    case departure if departure.status == "WriteOffNotification" =>
-      DepartureStatus("departure.status.writeOffNotification", actions = Nil)
+  private def writeOffNotification: PartialFunction[Departure, DepartureStatusViewModel] = {
+    case departure if departure.status == WriteOffNotification =>
+      DepartureStatusViewModel("departure.status.writeOffNotification", actions = Nil)
   }
 
+<<<<<<< HEAD:app/viewModels/DepartureStatusViewModel.scala
```

Comment:
Please remove this merge conflict auto comments

Reply:
Yea I'd spotted that after I raised the PR, it wasn't showing in the file main file - it somehow had created this file 'DepartureStatusViewModel.scala~HEAD' and it was showing in github but not in IntelliJ. I've removed it and seems okay now